students/johnpharmd/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,13 @@
+def fizzbuzz():
+    n = 1
+    while n < 101:
```

Comment:
Check out Python's built-in range function for a more concise alternative to the while loop:

`for x in range(1, 101):`